docs/getstarted/device.mdx

```diff
@@ -0,0 +1,189 @@
+import VSCode from "../_images/vscodehello.png";
+import JaguarVideo from "../_images/146210503-24811800-bb26-4244-817d-6422b20e6786.png";
+
+# Run on your device
+
+This short guide shows how to install Toit, flash the Toit firmware on your [ESP32](https://www.espressif.com/en/products/socs/esp32),
+setup your local development environment, and finally deploy a Toit application on your ESP32 over-the-air.
+
+It uses the open-source Jaguar service to enable live reloading when developing for the ESP32 and you
+can watch a short video that shows how you can experience Jaguar on your ESP32 in two minutes:
+
+[<img src={JaguarVideo} alt="Video presentation of Jaguar" />](https://youtu.be/cU7zr6_YBbQ)
```

Comment:
See comment for `docs/_images/14...`: might be nicer to use the youtube image instead. Unfortunately, it seems like the video is missing a maxresdefault option.

You might be able to generate it: https://stackoverflow.com/a/48178125